architectures

```diff
@@ -1,3 +1,4 @@
 bashbrew-arch   variants
 amd64     default,alpine,onbuild,slim,wheezy
 ppc64le default,onbuild,slim
+windows-amd64  windowsservercore,windowsservercore/onbuild,nanoserver,nanoserver/onbuild
```

Comment:
I used `windows-amd64` as it seems to be used for other official images, see https://github.com/docker-library/official-images/blob/master/README.md#architectures-other-than-amd64. It's a combination of OS + ARCH.